torch/testing/_internal/common_device_type.py

```diff
@@ -1094,6 +1094,11 @@ def skipCPUIfNoLapack(fn):
     return skipCPUIf(not torch._C.has_lapack, "PyTorch compiled without Lapack")(fn)
 
 
+# Skips a test on CPU if FFT is not available.
+def skipCPUIfNoFFT(fn):
```

Comment:
This name is a little odd because "NoFFT" suggests there's no fft available on any device type. It would more accurately be "skipCPUIfNoCPUFFT" or "skipCPUIfNoMKLorPocket" or something